Makefile

```diff
@@ -1,11 +1,11 @@
 all: webserver 
 
-webserver: main.cpp nginx-configparser/config_parser.cc server.cpp connection.cpp connection_manager.cpp request_handler.cpp reply.cpp mime_types.cpp request_parser.cpp
-	g++ -std=c++0x -I $(BOOST_PATH) main.cpp nginx-configparser/config_parser.cc server.cpp connection.cpp connection_manager.cpp request_handler.cpp reply.cpp mime_types.cpp request_parser.cpp -o webserver -lpthread -lboost_system \
+webserver: main.cpp nginx-configparser/config_parser.cc server.cpp connection.cpp connection_manager.cpp request_handler.cpp reply.cpp mime_types.cpp
```

Comment:
We could probably change this Makefile to be a little more concise using variables and the other things that were mentioned in class on Monday